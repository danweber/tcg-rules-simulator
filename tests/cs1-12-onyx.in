VERSION:2
# [Your Turn] When this Monster attacks, trash the top card of your opponent's security stack.
FIELD1:Attack1,CS1-12 Attack2,CS1-12 Attack3,CS1-12
SECURITY2:DW1-Van DW1-Van DW1-Van
GAME:P1 T2 MAIN 3
DUMP:SECURITY,TRASH

ECHO: -- Attack with first, two things end up removed
CMD:ATTACK Attack1 Player
DUMP:SECURITY,TRASH


ECHO: -- Attack with second, I win
CMD:ATTACK Attack2 Player
DUMP:SECURITY,TRASH
