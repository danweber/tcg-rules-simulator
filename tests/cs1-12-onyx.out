FIELD1:Attack1,CS1-Onyxshield Attack2,CS1-Onyxshield Attack3,CS1-Onyxshield
FIELD2:
SECURITY1:
SECURITY2:DW1-Vanillamon DW1-Vanillamon DW1-Vanillamon
TRASH1:
TRASH2:
GAME:P1 T2 MAIN 3
ECHO: -- Attack with first, two things end up removed
FIELD1:Attack1,CS1-Onyxshield,REST Attack2,CS1-Onyxshield Attack3,CS1-Onyxshield
FIELD2:
SECURITY1:
SECURITY2:DW1-Vanillamon
TRASH1:
TRASH2:DW1-Vanillamon DW1-Vanillamon
GAME:P1 T2 MAIN 3
ECHO: -- Attack with second, I win
FIELD1:Attack1,CS1-Onyxshield,REST Attack2,CS1-Onyxshield,REST Attack3,CS1-Onyxshield
FIELD2:
SECURITY1:
SECURITY2:
TRASH1:
TRASH2:DW1-Vanillamon DW1-Vanillamon DW1-Vanillamon
GAME:P1 T2 GAME_OVER 3
